demos/tabs/index.html

```diff
@@ -76,6 +80,29 @@
                         app.logger.log('triggered select item via pre select (manually)');
                         app.sandbox.emit('husky.tabs.contacts.item.clicked', event);
                     });
+
+                    var isTabChangeAllowed = function () {
+                        var allowed = $.Deferred();
+                        if (Math.floor(Math.random() * 2) >= 1) {
```

Comment:
Why not simply creating two different tabs? This way you would have to manually try to reload until you get the desire behavior to test.